### ISSUE TITLE
[Request] Update PyPi distribution

### ISSUE THREAD

**Issue by ClimbsRocks** [Microsoft/LightGBM#772]
Looks like there's been a bunch of great work done over the past month! Any chance we could get an updated build on PyPi so it's easy for everyone to take advantage of this?

Specifically, at my company, we want to use LightGBM on our production system, but the way we install dependencies relies on `pip install`ing. We're pretty reliant on categorical features, so I'm particularly interested in the optimal splits for categorical features that was built in a few weeks back. 

Thanks!

**Comment by guolinke** [comment]
@ClimbsRocks there is one small issue in python-package with categorical feature: https://github.com/Microsoft/LightGBM/issues/773 

will update pypi after fixed.

**Comment by ClimbsRocks** [comment]
Thanks! That'll be a differentiator from xgboost, which hasn't cut a new
release in a while.

**Comment by Laurae2** [comment]
@ClimbsRocks The current development xgboost version would create way more issues than adding features. Fast histogram is currently in an unstable state (you would need a version from 15 Feb 2017 to avoid OS-specific issues).

What would you need specifically from a newer xgboost?

**Comment by guolinke** [comment]
updated.
